Documentation/markdown/UnitTests.md

```diff
@@ -0,0 +1,24 @@
+# Unit Tests
+
+OIP already provides a set of unit tests to check continuously that even after changes 
+to the code, everything is still working smoothly.
+Primarily official buildingSMART IFC example files are used to check OIP for correctness
+since they come with screenshots and additional information on how a correct visualization needs to 
+look.
+In this way, specific IfcEntities can be inspected individually for correct implementation.
+
+Currently, the OIP unit tests are not automated and need to be run manually.
```

Comment:
There should be an open issue for that - reference maybe?

Reply:
I'm not able to find it

Reply:
#179